pkg/image/apis/image/helper.go

```diff
@@ -10,3 +15,19 @@ const (
 	// DockerDefaultV2Registry is the host name of the default v2 registry
 	DockerDefaultV2Registry = "registry-1." + DockerDefaultRegistry
 )
+
+// SplitImageSignatureName splits given signature name into image name and signature name.
+func SplitImageSignatureName(imageSignatureName string) (imageName, signatureName string, err error) {
```

Comment:
not allowed here.  Moving the imageutil package is ok, but no more helpers in api packages.

Reply:
i can make copies?

Reply:
copied made